internal/service/resourcepolicy/resource.go

```diff
@@ -35,6 +38,40 @@ type resourcePolicyRS struct {
 	config.RSCommon
 }
 
+var charSetAlphaNum = []rune("abcdefghijklmnopqrstuvwxyz012346789")
+
+// randPolicyName to workaround for POLICY_CANNOT_CONTAIN_A_DUPLICATE_NAME error until https://jira.mongodb.org/browse/CLOUDP-272944 is resolved.
```

Comment:
https://jira.mongodb.org/browse/CLOUDP-272944 is actually resolved, is it a wrong link?

Reply:
Thank you for spotting this.
Updating in a separate PR. Should be: https://jira.mongodb.org/browse/CLOUDP-274990

Reply:
Updated here: 8b40cbf490db508880ad539fa051610734c1a790